packages/linter/src/generators/workspace-rule/workspace-rule.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  applyChangesToString,
+  ChangeType,
+  convertNxGenerator,
+  formatFiles,
+  generateFiles,
+  joinPathFragments,
+  logger,
+  Tree,
+} from '@nrwl/devkit';
+import { camelize } from '@nrwl/workspace/src/utils/strings';
+import { join } from 'path';
+import * as ts from 'typescript';
+import { workspaceLintPluginDir } from '../../utils/workspace-lint-rules';
+import { lintWorkspaceRulesProjectGenerator } from '../workspace-rules-project/workspace-rules-project';
+
+export interface LintWorkspaceRuleGeneratorOptions {
+  name: string;
+  directory: string;
+}
+
+export async function lintWorkspaceRuleGenerator(
+  tree: Tree,
+  options: LintWorkspaceRuleGeneratorOptions
+) {
+  // Ensure that the workspace rules project has been created
+  await lintWorkspaceRulesProjectGenerator(tree);
```

Comment:
You need to return the task returned by this generator.

Currently, the npm install isn't triggered afterwards.

Reply:
Thanks, done